src/Network/Network/help/New-AzPrivateLinkServiceIpConfig.md

```diff
@@ -14,7 +14,7 @@ Create a private link service ip configuration.
 
 ```
 New-AzPrivateLinkServiceIpConfig -Name <String> [-PrivateIpAddressVersion <String>]
- [-PrivateIpAddress <String>] [-PublicIpAddress <PSPublicIpAddress>] [-Subnet <PSSubnet>]
```

Comment:
Did you regenerate docs with platyPS. I think it should've left PublicIpAddress there